lisa/sut_orchestrator/azure/features.py

```diff
@@ -710,7 +710,7 @@ def get_raw_data_disks(self) -> List[str]:
         pattern = re.compile(r"/dev/disk/azure/scsi[0-9]/lun[0-9][0-9]?", re.M)
         # /dev/disk/azure/scsi1/lun0
         cmd_result = self._node.execute(
-            "ls -d /dev/disk/azure/scsi*/*", shell=True, sudo=True
+            "ls -d /dev/disk/azure/scsi1/*", shell=True, sudo=True
```

Comment:
What's problem with original pattern? I saw Divyansh just changed the regexp few days ago. This change includes scsi0 as well. Can you update code comments to explain why filter only scsi1, not scsi*?

Reply:
in some distro, the os and temp disk display under /dev/disk/azure/scsi0, for data disks, they are under /dev/disk/azure/scsi1

Reply:
I think the change from Divyansh is to include scsi0, but with this change, scsi0 is excluded. How to handle it?

Reply:
There were disks which followed scsi0 indexing as well.

Reply:
I found the disks under scsi0 are os and temp disk, do you have an image which has data disks under /dev/disk/azure/scsi0

Reply:
Sorry I don't remember the image and VM size explicitly.

Reply:
@squirrelsc @somil55 this method is to get the data disks from /dev/disk/azure/scsi* folder
refer
https://docs.microsoft.com/en-us/troubleshoot/azure/virtual-machines/troubleshoot-device-names-problems#identify-disk-luns